cumulusci/tasks/apex/testrunner.py

```diff
@@ -236,6 +243,19 @@ def _init_options(self, kwargs):
         self.required_per_class_code_coverage_percent = int(
             self.options.get("required_per_class_code_coverage_percent", 0)
         )
+        # Raises a TaskOptionsError when the user provides both test_suite_names and test_name_match.
+        if (
+            self.options["test_suite_names"] is not None
+            and len(self.options["test_suite_names"]) > 0
```

Comment:
Because of the way truthy semantics work in Python, you can reduce these conditionals to `self.options["test_suite_names"]`. An empty list is falsy, as is `None`.

Reply:
Thanks for the valuable comment. Updated as requested.